### PR TITLE
[icons] feat: new icon "array-floating-point"

### PR DESCRIPTION
**Pull request by eugenechantk**
#### Checklist

- [ ] Includes tests
- [ ] Update documentation

#### Screenshot
Screen Shot 2022-07-05 at 4 57 45 PM

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @eugenechantk! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: packages/icons/icons.json [hunk 1]
```diff
@@ -3890,5 +3890,19 @@
...
+    {
+        "displayName": "Floating point array",
+        "iconName": "array-floating-point",
+        "tags": "number, float, double, decimal, array",
+        "group": "miscellaneous",
+        "codepoint": 62253
+    },
+    {
+        "displayName": "Cube: edit",
+        "iconName": "cube-edit",
```
Comment (adidahiya) [inline packages/icons/icons.json]: compilation is failing, it looks like you accidentally added this extra icon definition here. please remove it.

**Review by adidahiya — CHANGES_REQUESTED** [review]
icons SVGs look good otherwise

File: packages/icons/icons.json (hunk 1 above)
Comment (eugenechantk) [reply packages/icons/icons.json]: @adidahiya removed. Please review again, thanks 🙏

**Review by adidahiya — APPROVED** [review]
lgtm 👍🏽 

image

**Pull request merged**